python/hsfs/feature_group_commit.py

```diff
@@ -38,6 +42,8 @@ def __init__(
     @classmethod
     def from_response_json(cls, json_dict):
         json_decamelized = humps.decamelize(json_dict)
+        if len(json_decamelized["items"]) >= 1:
```

Comment:
there should be a count field returned by the API that lets you know how many items there are in the list. Check if we set the count field in the `CommitDTO` on the Hopsworks side.